app/packages/klogs/src/components/Logs.tsx

```diff
@@ -524,6 +525,15 @@ const Documents: FunctionComponent<{
                     >
                       {field}
                     </TableSortLabel>
+                    <IconButton
+                      edge="end"
+                      color="inherit"
+                      size={'small'}
+                      sx={{ m: 0 }}
+                      onClick={() => selectField?.(field)}
+                    >
+                      <RemoveCircleOutline fontSize="small" />
+                    </IconButton>
```

Comment:
core change.

im totally open to use any kind of other icon here or to add/change styling.